docs/docs/ias.md

```diff
@@ -1,41 +1,49 @@
 # Intel Attestation Service
 
-!!! important
-    You don't need to read and perform these steps to deploy a Conclave app, unless you wish to use older
-    hardware that isn't capable of using the DCAP protocol. [Azure Gen 2 Virtual Machines](azure.md) in particular do not
-    need any of these steps.
+!!!Note
+    
+    You don't need to perform these steps if you use [Azure Gen 2 VMs](machine-setup.md) or hardware that supports the
+    DCAP attestation protocol.
 
-How do you know a remote computer is fully up to date with the latest security patches and isn't physically compromised?
-This is the question Intel's Attestation Service (IAS) exists to answer. IAS is an online service that evaluates
-evidence from an enclave and returns:
+Intel's Attestation Service (IAS) helps to verify that a remote computer isn't physically compromised and has the 
```

Comment:
I would change `Intel's Attestation Service` with `Intel Attestation Service`, removing the possessive `'s`.

Reply:
Done.